heat/core/trigonometrics.py

```diff
@@ -99,6 +105,32 @@ def arctan(x, out=None):
     return local_op(torch.atan, x, out)
 
 
+def arctan2(x1, x2) -> dndarray.DNDarray:
+    """
+    Element-wise arc tangent of ``x1/x2`` choosing the quadrant correctly.
+    Returns a new :class:``DNDarray`` with the signed angles in radians between vector (``x2``,``x1``) and vector (1,0)
+
+    Parameters
+    ----------
+    x1 : :class:DNDarray
+         y-coordinates
+    x2 : :class:DNDarray
+         x-coordinates. If ``x1.shape != x2.shape``, they must be broadcastable to a common shape (which becomes the shape of the output).
```

Comment:
in Markdown code style, please refrain from using whitespaces around operators (it looks shitty in the docs due to the different font and size)